src/components/admin/ImportCSV.tsx

```diff
@@ -74,7 +77,16 @@ export default function ImportCSV() {
                                 </button>
                                 <ProgressBar className="progressBar me-2 mt-3" />
                             </div>
-
+                        </div>
+                        <div className="row">
+                            <div className="col d-flex p-0 my-2">
+                                <p style={{color: "white"}}>{
```

Comment:
Ujednolicenie sposobu stylowania.
```suggestion
                                <p className="theme-text-light">{
```